modules/products/bitbucket/variables.tf

```diff
@@ -132,4 +132,8 @@ variable "display_name" {
   description = "The display name of Bitbucket instance."
   type        = string
   default     = null
+  validation {
+    condition     = length(var.display_name) > 0 && length(var.display_name) < 255
```

Comment:
I think display name is not mandatory field could be null (we have null as default). Please add null as an accepted condition in the validation and update the error message.

Reply:
Good spot!

Reply:
One more thing, check `length(var.display_name) < 255`  is sufficient check here, we don't need to check if the length is more than 0 as empty string is still a valid input here.

Reply:
I don't think empty string is valid here:
<img width="620" alt="Screen Shot 2022-03-10 at 4 29 18 pm" src="https://user-images.githubusercontent.com/416238/157595880-4ab88f62-16b6-4822-a984-98734654c3cb.png">